Patch.py

```diff
@@ -162,102 +154,11 @@ def patch(self, target: str):
             f.write(result)
 
 
-# legacy patch handling follows:
 GAME_ALTTP = "A Link to the Past"
 GAME_SM = "Super Metroid"
 GAME_SOE = "Secret of Evermore"
 GAME_SMZ3 = "SMZ3"
 GAME_DKC3 = "Donkey Kong Country 3"
```

Comment:
We should try to get rid of those as well, but that's probably out of scope here, since that'd be SNIClient rework.

Reply:
agreed on both. So that's more towards @PoryGone 's scope right now.